lib/ReportHistoryStore.jsx

```diff
@@ -263,9 +263,6 @@ export default class ReportHistoryStore {
             delete this.cache[reportID];
         }
 
-        // We'll poll the API for the un-cached history
-        const cachedHistory = this.cache[reportID] || [];
-
```

Comment:
This change is connected to TS migration?

Reply:
It was unused, so there was an ESLint error